src/scripts/main.js

```diff
@@ -1,3 +1,6 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.top = 'calc(50% - 25px)';
```

Comment:
The spider position is hardcoded, it's better to use DOM element properties to calculate its position.